src/components/application_manager/test/resumption/resume_ctrl_test.cc

```diff
@@ -647,6 +649,15 @@ TEST_F(ResumeCtrlTest, StartResumption_AppWithSubscriptionToWayPoints) {
   EXPECT_CALL(
       mock_app_mngr_,
       SubscribeAppForWayPoints(A<application_manager::ApplicationSharedPtr>()));
+  EXPECT_CALL(*application_manager::MockMessageHelper::message_helper_mock(),
+              CreateMessageWithFunctionID(
+                  _, hmi_apis::FunctionID::Navigation_SubscribeWayPoints))
+      .WillOnce(Return(message_to_hmi));
+  ON_CALL(mock_app_mngr_, GetRPCService())
+      .WillByDefault(ReturnRef(mock_rpc_service_));
+  EXPECT_CALL(mock_rpc_service_, ManageHMICommand(message_to_hmi, _))
```

Comment:
command source probably should be checked.

Reply:
Done in da940c2e0e2ea1d863b840c2a9204159e0d6bfc9